src/content/ui-api/kubectl-gs/get-appcatalogs.md

```diff
@@ -0,0 +1,112 @@
+---
+linkTitle: get appcatalogs
+title: "'kubectl gs get appcatalogs' command reference"
+description: Reference documentation on how to list appcatalogs and get details for a single appcatalog using 'kubectl gs'.
+weight: 20
+menu:
+  main:
+    parent: uiapi-kubectlgs
+aliases:
+  - /reference/kubectl-gs/get-appcatalogs/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+user_questions:
+  - How can I list appcatalogs using kubectl?
+  - How can I inspect appcatalogs using kubectl?
+---
+
+# `kubectl gs get appcatalogs`
+
+Like with all `get` commands in `kubectl`, this command can be used to get details on one item, an [AppCatalog]({{< relref "/ui-api/management-api/crd/appcatalogs.application.giantswarm.io.md" >}})
+custom resource in this case, or list several of them.
+
+## Usage
+
+### Get a list of App Catalogs {#list}
+
+Simply execute
+
+```nohighlight
+kubectl gs get appcatalogs
+```
+
+to list some information on all available public App Catalogs.
+
+Here is some example output:
+
+```nohighlight
+NAME                    CATALOG URL                                                   AGE
+giantswarm              https://giantswarm.github.io/giantswarm-catalog/              280d
+giantswarm-playground   https://giantswarm.github.io/giantswarm-playground-catalog/   280d
+```
+
+### Get specific App Catalog
+
+When used with an App Catalog name as additional argument, the command will show
+the latest available version of each App in the catalog according to
+[semantic versioning](https://semver.org/).
+
+Example:
+
+```nohighlight
+kubectl gs get appcatalogs giantswarm
+```
+
+```nohighlight
+kubectl gs get appcatalogs giantswarm
+CATALOG      APP NAME                       APP VERSION   VERSION                                          AGE
+giantswarm   cert-manager-app               1.3.1         2.7.0                                            25h
+```
+
+Note: As an alternative to `get appcatalogs`, `get appcatalog` will also work.
+
+## Output {#columns}
+
+The standard tabular output format for App Catalogs features these columns:
+
+- `NAME`: Name of the App Catalog.
+- `URL`: URL for the Helm chart repository.
+- `AGE`: How long ago the appcatalog was created.
```

Comment:
Question for me:
Does `AGE` here refer to "we established this 280 days ago" or "we created this particular CR 280 days ago"

Reply:
Yes, that's not very clear.

It maps to the creationTimestamp of the appcatalog CR. I've renamed it to `CREATED`. As I think that's more accurate and aligns with `get clusters`.